testsuite/test-runner/src/main/java/org/wildfly/core/testrunner/Server.java

```diff
@@ -221,6 +256,66 @@ private void safeCloseClient() {
     }
 
     /**
+     * Reload current server
+     * This method makes sure client on server is still operational after reload is done.
+     * @param adminOnly tells server to boot in admin only mode or normal
+     * @param timeout time in miliseconds to wait for server to come back after reload
+     */
+    public void reload(boolean adminOnly,int timeout){
+        executeReload(adminOnly);
+        waitForLiveServerToReload(timeout); //30 seconds
```

Comment:
It wasn't there at time of initial writing of this code :-) 
But in any case this wouldn't work as this is test runner that has no dependencies to shared testsuite. If anything ServerReload in testsuite should depend on test runner's code.

Also most of tests in core should only use test runner and its classes for reload capabilities, instead of relying on yet another utility class for this.